test/framework/toy_build.py

```diff
@@ -563,23 +565,64 @@ def test_toy_gid_sticky_bits(self):
             self.assertTrue(perms & stat.S_ISVTX, "sticky bit set on %s" % fullpath)
 
     def test_toy_group_check(self):
-        """Test the userInGroup function with lua modules"""
-        toy_ec_file = os.path.join(os.path.dirname(__file__), 'easyconfigs', 'test_ecs', 't', 'toy', 'toy-0.0-group.eb')
+        """Test presence of group check in generated (Lua) modules"""
+        fd, dummylogfn = tempfile.mkstemp(prefix='easybuild-dummy', suffix='.log')
+        os.close(fd)
+
+        # figure out a group that we're a member of to use in the test
+        out, ec = run_cmd("groups $USER", simple=False)
+        self.assertEqual(ec, 0, "Failed to select group to use in test")
+        group_name = out.split(' ')[0]
```

Comment:
`groups $USER` returns this string: `$USER : $GROUPS` (with expanded environment variables). `group_name` would be then `$USER` and the test would fail (if `$USER` is defined). I think we can simply use `groups` in `run_cmd`. Then the output would be fine `$USER : ` is not prepended.